api/sql/userSQL/initialize.sql

```diff
@@ -1,9 +1,18 @@
 DROP TABLE IF EXISTS Users CASCADE;
+DROP TABLE IF EXISTS UserSessions CASCADE;
 
 CREATE TABLE Users (
   id SERIAL PRIMARY KEY,
   username TEXT NOT NULL,
   password TEXT NOT NULL,
   data_id INT NOT NULL,
   FOREIGN KEY (data_id) REFERENCES UserData (id)
+);
+
+CREATE TABLE UserSessions (
+  id SERIAL PRIMARY KEY,
+  session_key VARCHAR (100) NOT NULL,
+  user_id INT NOT NULL,
+  exp_date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP + interval '4 hours',
+  FOREIGN KEY (user_iD) REFERENCES Users (id)
```

Comment:
Does `user_iD` need to be `user_id`?

Reply:
SQL with interpret it all lowercase. Should be changed for consistency purposes though